wcc-contentful-app/app/controllers/wcc/contentful/app/contact_form_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class WCC::Contentful::App::ContactFormController < ApplicationController
+  def create
+    id = params['id']
+    raise ArgumentError, 'missing form ID' unless id
+
+    form_model = WCC::Contentful::Model.find(id)
+    to_email = form_model.notificationEmail
+    data = {}
+
+    form_model.fields.each do |item|
+      data[item.title] = params[item.title]
+    end
+
+    UserMailer.contact_form_email(to_email, data).deliver_later
+
+    connected =
```

Comment:
Can we memoize this, so we don't have to check it every time?

I assume the purpose of this is to check if the app has a postgres instance, correct?

Reply:
Right, that's the only purpose of it. But yeah, I'll try to memoize it.